terraform/modules/content_delivery_network/main.tf

```diff
@@ -455,6 +455,31 @@ resource "aws_cloudfront_distribution" "tiles" {
     }
   }
 
+  ordered_cache_behavior {
```

Comment:
Comment at line 426 is confusing.  Should that comment be here right above line 458, and the comment at line 426 should be "Send all cog requests to the tile cache app"?

Reply:
Both are titiler endpoints. I have added clarifying comment for the DIST alerts one I just added and updated the generic one.